blocks/card-carousel/card-carousel.css

```diff
@@ -0,0 +1,220 @@
+.card-carousel {
+  --card-width: 16rem;
+
+  margin-top: 3rem;
+  padding-left: 0.5rem;
+}
+
+.career-slider {
+  display: flex;
+  flex-direction: column;
+  margin-left: 12rem;
+}
+
+.cards-card-image {
+  height: 12.5rem;
+  margin: 0;
+  padding: 0;
+}
+
+.career .career-slider{
+  margin-left: -1rem;
+}
+
+.career-slides {
+  display: flex;
+  overflow-x: scroll;
+  scroll-behavior: smooth;
+
+  /* We want to hide the scroll bar */
+  -ms-overflow-style: none; /* for Internet Explorer, Edge */
```

Comment:
I think edge supports `::-webkit-scrollbar`
https://caniuse.com/?search=-webkit-scrollbar
And imho we can drop the internet explorer support